src/math/exp_fcn_hifi.c

```diff
@@ -26,10 +27,13 @@
 #include <xtensa/tie/xt_hifi3.h>
 #endif
 
+#include <xtensa/tie/xt_hifi2.h>
+#include <xtensa/tie/xt_FP.h>
+
 #define SOFM_CONVERG_ERROR 28823037624320LL /* error smaller than 1e-4,1/2 ^ -44.7122876209085 */
 #define SOFM_BIT_MASK_LOW_Q27P5 0x0000000008000000
 #define SOFM_BIT_MASK_Q62P2 0x4000000000000000LL
-#define SOFM_QUOTIENT_SCALE BIT(30)
+#define SOFM_QUOTIENT_SCALE 0x40000000
 #define SOFM_TERMS_Q23P9 0x800000
 #define SOFM_LSHIFT_BITS 0x2000
 /*
```

Comment:
Btw, these macros look similar (the same ?) as macros in the previous source file. If so, we should declare them in a exp header.